### PR TITLE
remove some typos from the documentation

### PR DESCRIPTION
**Pull request by PatWie**
add probably add some new ones :wink:

### CODE REVIEW

**Comment by ppwwyyxx** [comment]
Actually "datapoint" was the term I planned to use, on anything yielded by a dataflow. It might not be a good name though..

**Comment by PatWie** [comment]
Ok I will restore changes belonging to `datapoint`.

File: tensorpack/dataflow/imgaug/crop.py [hunk 1]
```diff
@@ -91,7 +91,7 @@ def perturb_BB(image_shape, bb, max_pertub_pixel,
     Args:
         image_shape: [h, w]
         bb (Rect): original bounding box
-        max_pertub_pixel: pertubation on each coordinate
+        max_pertub_pixel: perturbation on each coordinate
```
Comment (PatWie) [inline tensorpack/dataflow/imgaug/crop.py]: Renaming `max_pertub_pixel`, too?

Comment (ppwwyyxx) [inline tensorpack/dataflow/imgaug/crop.py]: OK!

**Comment by ppwwyyxx** [comment]
Great! Is that all?

**Comment by PatWie** [comment]
I will do another pass tomorrow or so.

**Comment by PatWie** [comment]
Probably it is better to merge this now to prevent manually resolving merge conflicts.

**Pull request merged**